src/main/java/frc/robot/subsystems/Hopper.java

```diff
@@ -0,0 +1,56 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.LoadingConstants;
+
+public class Hopper extends SubsystemBase {
+
+  private final TalonSRX m_hopperMotor;
+
+  /**
+   * Creates a new Hopper.
+   */
+  public Hopper() {
+    m_hopperMotor = new TalonSRX(LoadingConstants.kLoadingMotorPort);
+  
+    motorInit(m_hopperMotor);
+    
+  }
+
+  public void motorInit(TalonSRX motor) {
+    motor.configFactoryDefault(); // configures the factory defaults in case and changes were previously made to the motor  
+  }
+
+  // starts the hopper motor to transfer the power cells from the intake to the cellavator
+  public void startHopperMotor(double speed) {
+    m_hopperMotor.set(ControlMode.Current, speed);
```

Comment:
This function parameter is a little misleading. Since the control mode is current, you don't pass a "speed" from -1.0 - 1.0.
```suggestion
  /**
   * Start the hopper motor to transfer the power cells from the intake to the cellavator.
   *
   * @param amperage Amperage target for the hopper motor.
   */
  public void startHopperMotor(double amperage) {
    m_hopperMotor.set(ControlMode.Current, amperage);
```

Reply:
In fact, unless we have a specific reason to use current or voltage, we should not be using the ControlMode parameter at all, since the default is percentage.  My suggestion for this would be

```suggestion
/**
 * Start the hopper motor to transfer the power cells from the intake to the cellavator.
 *
 * @param amperage Amperage target for the hopper motor.
 */
 public void runHopperMotor (double speed) {
 m_hopperMotor.set(speed);
```

Reply:
It gives an error if I do not put a control mode, so I set it to percent output.